src/query/workerQueue.js

```diff
@@ -244,7 +244,6 @@ class WorkerQueue {
         if (this.dht._isSelf(closer.id)) {
           return
         }
-        closer = this.dht.peerStore.put(closer)
```

Comment:
No new information of the peer at this point

Reply:
Also it's unnecessary as the emit below will cause libp2p to update the peer store